app/src/main/kotlin/cz/covid19cz/erouska/ui/dashboard/DashboardFragment.kt

```diff
@@ -236,14 +237,19 @@ class DashboardFragment : BaseFragment<FragmentPermissionssDisabledBinding, Dash
     }
 
     private fun isPlayServicesObsolete(): Boolean {
-        val current = PackageInfoCompat.getLongVersionCode(
-            requireContext().packageManager.getPackageInfo(
-                GoogleApiAvailability.GOOGLE_PLAY_SERVICES_PACKAGE,
-                0
+        return try {
+            val current = PackageInfoCompat.getLongVersionCode(
+                requireContext().packageManager.getPackageInfo(
+                    GoogleApiAvailability.GOOGLE_PLAY_SERVICES_PACKAGE,
+                    0
+                )
             )
-        )
 
-        return current < AppConfig.minGmsVersionCode
+            current < AppConfig.minGmsVersionCode
+        } catch (e: Exception) {
+            L.e(e)
+            false
```

Comment:
nemělo by tady být true?

Reply:
Asi jo, pokud použijeme logiku nemáPlayServices -> isObsolete, tak ano

Reply:
Bez Play Services nemůže aplikace fungovat, proto bych zobrazil ten screen

Reply:
Už to tam je